scss/partials/portfolio.scss

```diff
@@ -0,0 +1,236 @@
+@use "variables";
+@use "reset";
+
+@import url('https://fonts.googleapis.com/css2?family=Baskervville:ital@0;1&family=Nunito:ital,wght@0,200..1000;1,200..1000&display=swap');
+
+button,
+label {
+    cursor: pointer;
+}
+
+.portfolio {
+    text-align: center;
+    margin-bottom: 42px;
+    .page-header {
+        padding: 46px 0 40px;
+        h2 {
+            font-family: "Baskervville", serif;
+            font-size: 40px;
+            line-height: 40px;
+            text-transform: uppercase;
+            color: variables.$text-darkbrown;
+            position: relative;
+            padding-bottom: 28px;
+            margin-bottom: 28px;
+            font-weight: 400;
+            letter-spacing: 6px;
+            &::after {
+                content: "";
+                display: block;
+                position: absolute;
+                left: 50%;
+                transform: translateX(-50%);
+                bottom: 0;
+                width: 139px;
+                height: 1px;
+                background-color: #E3B6B1;
+            }
+        }
+        p {
+            font-size: 16px;
+            font-family: "Nunito", sans-serif;
+            padding: 0 2rem;
+            font-weight: 200;
+            line-height: 28px;
+        }
+    }
+    .page-tabs {
+        margin-bottom: 24px;
+        width: 100%;
+        max-width: 335px;
+        ul {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            .page-tab {
+                border-bottom: 1px solid #888;
+                position: relative;
+                width: 50%;
+                &::after {
+                    content: "";
+                    display: block;
+                    width: 100%;
+                    height: 1px;
+                    position: absolute;
+                    left: 0;
+                    bottom: -1px;
+                    transition: all .3s ease-in-out;
+                    z-index: 1;
+                }
+                &.on {
+                    .btn-tab {
+                        color: #181818;
+                    }
+                    &::after {
+                        background-color: #181818;
+                    }
+                }
+                // &:last-child.on {
```

Comment:
Is there any intention in leaving this code
if you do not need this code, please delete (if possible)
Not big problem, so we can ignore.
but deleting is better

Reply:
There is no reason, I should check and make to clean before creating pull request.
I committed it erasered comment parts.